mobile/src/main/java/com/gelakinetic/mtgfam/helpers/database/CardDbAdapter.java

```diff
@@ -533,6 +535,12 @@ public static Cursor fetchCards(long[] ids, String orderByStr, SQLiteDatabase da
         }
     }
 
+    public static Cursor fetchCardByName(String name, List<String> fields, boolean shouldGroup,
+                                         boolean offlineOnly, boolean preferOptionalFoil, SQLiteDatabase mDb)
+            throws FamiliarDbException {
+        Set<String> languages = new HashSet<>(Arrays.asList(new String[]{"en"}));

```

Comment:
Why isn't this one grabbing the preference?

Reply:
Also copying the javadoc would be nice.

Reply:
OK, it wasn't clear to me from just this diff that this function was only used internally. I double checked it and it's good. A javadoc note is enough here, you don't need to change the function name.

Unrelated, you don't have to explicitly declare the array here. This works just as well:
```
Set<String> languages = new HashSet<>(Arrays.asList("en"));
```

Reply:
I had a look on the overloaded functions again and decided to remove these implementations.
I think it's clearer to explicitly state which languages the search functions are called for.
To keep the function calls as before only makes it easier to get mixed up, when you need them in the future.

Reply:
> Unrelated, you don't have to explicitly declare the array here. This works just as well:
> ```
> Set<String> languages = new HashSet<>(Arrays.asList("en"));
> ```
Of course, you're right.